bucket/googlechrome.json

```diff
@@ -39,6 +39,9 @@
         "    Copy-Item \"$env:LocalAppData\\Google\\Chrome\\User Data\\*\" \"$dir\\User Data\" -Recurse",
         "}"
     ],
+    "env_set": { 
```

Comment:
```suggestion
    "env_set": {
```

Reply:
Oh sorry, I didn't check this and just commit your suggestion. Wow, I never noticed that the testing even check this (is this the reason it fails?).